app/src/main/java/com/github/kulebin/myfoursquareapp/ContextHolder.java

```diff
@@ -0,0 +1,19 @@
+package com.github.kulebin.myfoursquareapp;
+
+import android.content.Context;
+
+public enum ContextHolder {
```

Comment:
be carefull when use such ContextHolder, especially in threads, and using for UI when activity is destroyed